clusterloader2/pkg/measurement/common/scheduler_latency.go

```diff
@@ -233,9 +242,10 @@ func (s *schedulerLatencyMeasurement) sendRequestToScheduler(c clientset.Interfa
 }
 
 type schedulingMetrics struct {
-	PredicateEvaluationLatency  measurementutil.LatencyMetric `json:"predicateEvaluationLatency"`
-	PriorityEvaluationLatency   measurementutil.LatencyMetric `json:"priorityEvaluationLatency"`
-	PreemptionEvaluationLatency measurementutil.LatencyMetric `json:"preemptionEvaluationLatency"`
-	BindingLatency              measurementutil.LatencyMetric `json:"bindingLatency"`
-	E2eSchedulingLatency        measurementutil.LatencyMetric `json:"e2eSchedulingLatency"`
+	PredicateEvaluationLatency         measurementutil.LatencyMetric `json:"predicateEvaluationLatency"`
+	PriorityEvaluationLatency          measurementutil.LatencyMetric `json:"priorityEvaluationLatency"`
+	PreemptionEvaluationLatency        measurementutil.LatencyMetric `json:"preemptionEvaluationLatency"`
+	BindingLatency                     measurementutil.LatencyMetric `json:"bindingLatency"`
+	E2eSchedulingLatency               measurementutil.LatencyMetric `json:"e2eSchedulingLatency"`
+	SchedulingAlgorithmDurationLatency measurementutil.LatencyMetric `json:"schedulingAlgorithmDurationLatency"`
```

Comment:
lets just name it ```SchedulingLatency```

Reply:
Once you rename it could you also document it (e.g. what @ahg-g wrote is good, that it tracks scheduling latency without binding)

Reply:
Document in the commit message and as a comment over the new field name